chrome-helper/eos-google-chrome.in

```diff
@@ -1,3 +1,10 @@
-#!/bin/sh
-prefix=@prefix@
+#!/bin/bash
+
+ABSOLUTE_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)/$(basename "${BASH_SOURCE[0]}")"
+
+if ! malcontent-client check ${ABSOLUTE_PATH} >/dev/null; then
```

Comment:
Same review comment as in the Chromium plugin again.

Reply:
Updated, same comments as in the chromium PR regarding --quiet.

Reply:
Updated again to support --quiet.